contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/standard_metrics.py

```diff
@@ -45,6 +49,34 @@ def get_available_memory_metric():
     return gauge
 
 
+def get_process_private_bytes():
+    try:
+        process = psutil.Process()
+        return process.memory_info().rss
+    except psutil.NoSuchProcess as ex:
```

Comment:
Curious, when would `NoSuchProcess` get triggered?

Reply:
`os.getpid` is called when constructing a Process with no pid. `NoSuchProcess` is raised if the pid does not exist. When accessing methods of the Process class, it is good to always catch `NoSuchProcess` and `AccessDenied`

Reply:
I'm very confused - are we getting the info from the current process? How could we get `NoSuchProcess`?

Reply:
Yes we are getting the info from the current process so there is no way to get `NoSuchProcess`. I included the exception based off of psutil docs to always include these two exception checks.
https://psutil.readthedocs.io/en/latest/#psutil.Process.memory_info